### PR TITLE
docs: More renames for wapi 

### PR DESCRIPTION
**Pull request by deep-volue**

### CODE REVIEW

File: docs/events.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ Listening for changes
 =====================
 
 
-WAPI supports listening for changes to curves.  An event listener is set up, and behaves
+VolueInsight Time-Series API supports listening for changes to curves.  An event listener is set up, and behaves
```
Comment (stigalvestad) [inline docs/events.rst:7]:
```suggestion
Volue Insight time series API supports listening for changes to curves.  An event listener is set up, and behaves
```

**Review by stigalvestad — CHANGES_REQUESTED** [review]
Good! See my comment on the naming. 
Can you also correct the double https://https// here and include that in this PR?

**Comment by deep-volue** [comment]
> Good! See my comment on the naming. Can you also correct the double https://https// here:
> 
> https://github.com/volueinsight/volue-insight-timeseries/blob/610df35b2d57bbc5d3514f340d17e1e9676ef5ca/docs/quickstart.rst?plain=1#L65
> 
> and include that in this PR?

Yes, I removed that double https

**Review by stigalvestad — APPROVED** [review]
Great, this is good!

**Pull request merged**